packages/backend-for-frontend/src/services/selfcareService.ts

```diff
@@ -121,12 +122,11 @@ export function selfcareServiceBuilder(
 
       const selfcareId = tenant.selfcareId;
       const users: selfcareV2ClientApi.UserResource[] =
-        await selfcareV2Client.getInstitutionProductUsersUsingGET({
+        await selfcareV2InstitutionClient.getInstitutionUsersByProductUsingGET({
           params: { institutionId: selfcareId },
           queries: {
             userId,
-            userIdForAuth: requesterId,
-            productRoles: roles,
+            productRoles: roles[0], // At this point the type has changed from string[] to string, so the query should only return one value
```

Comment:
```suggestion
            productRoles: roles.join(",")
```
Shouldn't this be a comma separated string?